Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -1121,7 +1135,10 @@ extension WebGPU.CommandEncoder {
         if let attachment = wgpuGetRenderPassDescriptorDepthSencilAttachment(descriptorSpan)?[0] {
```

Comment:
Existing nit: this function has a typo in its name.

Reply:
I will fix the naming in this PR since its only used in a limited number of places